### PR TITLE
Fix misunderstanding sentence in Value Combiners documentation

### PR DESCRIPTION
**Pull request by hieuwu**
### :sparkles: What kind of change does this PR introduce? (Bug fix, feature, docs update...)
Docs up date

### :arrow_heading_down: What is the current behavior?
Misunderstanding the sentence

### :new: What is the new behavior (if this is a feature change)?
Rewrite the sentence

### :boom: Does this PR introduce a breaking change?
No

### :bug: Recommendations for testing
No

### :memo: Links to relevant issues/docs
https://www.mvvmcross.com/documentation/fundamentals/value-combiners

### :thinking: Checklist before submitting

- [x] All projects build
- [x] Follows style guide lines ([code style guide](https://github.com/MvvmCross/MvvmCross#code-style-guidelines))
- [x] Relevant documentation was updated ([docs style guide](https://www.mvvmcross.com/documentation/contributing/mvvmcross-docs-style-guide))
- [x] Rebased onto current develop

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/MvvmCross/MvvmCross?pullRequest=4121) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/MvvmCross/MvvmCross?pullRequest=4121) before we can accept your contribution.<br/><br/>:x: hieuwu [sign now](https://cla.dotnetfoundation.org/MvvmCross/MvvmCross?pullRequest=4121)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/MvvmCross/MvvmCross?pullRequest=4121) it.</sub>

File: docs/_documentation/fundamentals/value-combiners.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Tibet binding (see [wiki/Databinding](https://www.mvvmcross.com/documentation/fu
 
          local:MvxBind="Text Format('{0} {1} {2}', Greeting(Gender), FirstName, LastName)"
          
-The rules and mechanisms for registering ValueCombiners are similar to those for registering ValueConverters. However, because combiners are not commonly declared in user code, MvvmCross doesn't current perform a Reflection sweep across your Core or UI assemblies. If you do want to add is used by default on the , then a ValueCombiner class named `FooValueCombiner` will be registered under the name `Foo`.
+The rules and mechanisms for registering ValueCombiners are similar to those for registering ValueConverters. However, because combiners are not commonly declared in user code, MvvmCross doesn't current perform a Reflection sweep across your Core or UI assemblies. If you do want to add by default, then a ValueCombiner class named `FooValueCombiner` will be registered under the name `Foo`.
```
Comment (Hackmodford) [inline docs/_documentation/fundamentals/value-combiners.md:14]: I think I found a typo

MvvmCross doesn't `current` perform

should be

MvvmCross doesn't currently perform

Comment (hieuwu) [reply docs/_documentation/fundamentals/value-combiners.md:14]: Yeah, thank for your addition :D

**Review by hieuwu — COMMENTED** [review]
Just review and fix your addition typo error

**Review by Cheesebaron — APPROVED** [review]

**Pull request merged**